MetaMask/metamask-extension
# Add blockaid related event parameters to signature request metrics

## **Description**
Fixes #20859

Blockaid related event parameters were missed out to be added to metrics for signature requests. The PR fixes this.

## **Manual testing steps**
1. Enable SEGMENT_FLASK_WRITE_KEY
2. Build flask
3. Open MM
4. Trigger any signature from the test dapp (malicious, non malicious, etc)
5. Check either the background console for the Segment request or your Segment project if you have set it up
6. Ensure that field related to PPOM is displayed

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

_[screenshot]_

### **After**

_[screenshot]_

## **Related issues**

fixes #20859

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
